src/main/java/eu/tsystems/mms/tic/testerra/plugins/xray/synchronize/DefaultSummaryMapper.java

```diff
@@ -25,7 +25,7 @@ public JqlQuery queryTest(MethodContext methodContext) {
 
     @Override
     public boolean shouldCreateNewTest(MethodContext methodContext) {
-        return true;
+        return false;
```

Comment:
Why changing the default behaviour here?

Reply:
Hi, like I described I reduced the risk to overwrite attributes, test steps, ... of existing test cases if you only want to map your tests via method name <-> issue summary.
In a similar Xray connector setup this was happend in the past.
Now the `DefaultSummaryMapper` only provides the mapping feature via name, not more. All other have to be activated _manually_.